examples/fuse_examples/multimodality/ehr_transformer/main_train.py

```diff
@@ -205,14 +195,12 @@ def train(
             pred="model.logits.gender",
             target="Gender",
             callable=F.cross_entropy,
-            preprocess_func=filter_gender_label_unknown_for_loss,
+            preprocess_func=filter_gender_label_unknown,
             weight=gender_loss_weight,
         )
 
-        train_metrics["gender_auc"] = Filter(
-            MetricAUCROC(pred="model.output.gender", target="Gender"),
-            "filter",
-            pre_collect_process_func=filter_gender_label_unknown_for_metric,
```

Comment:
self-reminder:

open an issue on the `pre_collect_process_func` that cause to disproportionate amount of NDict's `__init__` calls.